client/reader/stream/index.jsx

```diff
@@ -173,6 +175,10 @@ class ReaderStream extends React.Component {
 		}
 	}
 
+	handleOpenSelectionNewTab = () => {
+		window.open( this.props.selectedPostKey.url, '_blank' );
```

Comment:
we should also use `noreferrer,noopener` as the feature string to prevent opened windows from [getting a handle back to us via window.opener](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Window_functionality_features).
```suggestion
		window.open( this.props.selectedPostKey.url, '_blank', 'noreferrer,noopener' );
```